setup.py

```diff
@@ -48,12 +47,13 @@
 ]
 
 install_requires = [
-    'Flask>=0.11.1',
-    'Flask-Admin>=1.5.3',
-    'Flask-Login>=0.3.0',
+    'Flask>=1.0.4'
```

Comment:
should this be ``invenio-base``? (according to the title?)

Reply:
yes, I changed it many times, I thought I will just wait until we align :)